modules/fedimint-ln-tests/tests/tests.rs

```diff
@@ -908,6 +914,19 @@ mod fedimint_migration_tests {
                             );
                             info!("Validated MetaOverrides");
                         }
+                        fedimint_ln_client::db::DbKeyPrefix::LightningGateway => {
+                            let gateways = dbtx
+                                .find_by_prefix(&LightningGatewayKeyPrefix)
+                                .await
+                                .collect::<Vec<_>>()
+                                .await;
+                            let num_gateways = gateways.len();
+                            ensure!(
+                                num_gateways > 0,
+                                "validate_migrations was not able to read any LightningGateways"
+                            );
+                            info!("Validated LightningGateways");
+                        }
```

Comment:
I guess this is why the DB snapshot needed to be regenerated. Before the test was just missing.